M365/Check-MailboxExtendedProperty.ps1

```diff
@@ -0,0 +1,133 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+#Requires -Modules @{ ModuleName="ExchangeOnlineManagement"; RequiredVersion="3.4.0" }
+
+<#
+.SYNOPSIS
+    Checks what mailbox extended properties (aka named properties) exist in the mailbox and if they are near to any limits.
+
+.DESCRIPTION
+    This script retrieves the mailbox extended properties for a specified user identity.
+
+.PARAMETER Identity
+    The identity of the user whose mailbox extended properties are to be retrieved.
+
+.PARAMETER Threshold
+    The quota threshold to check for having exceeded. Default is 0.9, which is 90% of the allowed quota.
+
+.PARAMETER SelectFirst
+    The number of sorted descending results to select, when checking any namepace or same name prefix quota. Default is 10.
+
+.EXAMPLE
+    .\Check-MailboxExtendedProperty.ps1 -Identity fred@contoso.com
+    .\Check-MailboxExtendedProperty.ps1 -Identity fred@contoso.com -Threshold 0.95
+    .\Check-MailboxExtendedProperty.ps1 -Identity fred@contoso.com -Threshold 0.7 -SelectFirst 20
+
+#>
+param(
+    [Parameter(Mandatory = $true, Position = 0)]
+    $Identity,
+    [Parameter(Mandatory = $false, Position = 1)]
+    $Threshold = 0.9,
```

Comment:
Let's have a `ValidateScript` here to make sure the value is between `.5` and less than `1`. Or whatever that makes sense here. But we don't want to have an end user provide something like `90` when they really wanted `.9`.

Reply:
Added parameter validation